share/harvesters/org_socialscienceregistry.py

```diff
@@ -0,0 +1,59 @@
+import csv
+import datetime
+
+import logging
+
+from share.harvest import BaseHarvester
+
+logger = logging.getLogger(__name__)
+
+
+class SCHarvester(BaseHarvester):
+    """
+    """
+    VERSION = 1
+
+    def do_harvest(self, start_date, end_date):
+        end_date = end_date.date()
+        start_date = start_date.date()
+        logger.info('Harvesting the social science registry %s - %s', start_date, end_date)
+        return self.fetch_records(start_date, end_date)
+
+    def fetch_records(self, start_date, end_date):
+
+        csv_download = self.requests.get(self.kwargs['csv_url'])
+
+        csv_download.raise_for_status()
+
+        decoded_content = csv_download.content.decode('utf-8')
+
+        cr = csv.reader(decoded_content.splitlines(), delimiter=',')
```

Comment:
You might want to consider using a [DictReader](https://docs.python.org/3/library/csv.html#csv.DictReader) and just sending back each dictionary.